pillar/edx/ansible_vars/ecommerce.sls

```diff
@@ -58,15 +58,15 @@ edx:
     ECOMMERCE_OSCAR_FROM_EMAIL: 'oscar@example.com' # TODO: Determine appropriate value for this setting 2019-01-15 TMM
 
     # CyberSource related
-    ECOMMERCE_CYBERSOURCE_PROFILE_ID: 'SET-ME-PLEASE'
-    ECOMMERCE_CYBERSOURCE_MERCHANT_ID: 'SET-ME-PLEASE'
-    ECOMMERCE_CYBERSOURCE_ACCESS_KEY: 'SET-ME-PLEASE'
-    ECOMMERCE_CYBERSOURCE_SECRET_KEY: 'SET-ME-PLEASE'
-    ECOMMERCE_CYBERSOURCE_SOP_ACCESS_KEY: 'SET-ME-PLEASE'
-    ECOMMERCE_CYBERSOURCE_SOP_PROFILE_ID: 'SET-ME-PLEASE'
-    ECOMMERCE_CYBERSOURCE_SOP_SECRET_KEY: 'SET-ME-PLEASE'
+    ECOMMERCE_CYBERSOURCE_PROFILE_ID: __vault__::secret-{{ business_unit }}/{{ environment }}/cybersource>data>profile_id
+    ECOMMERCE_CYBERSOURCE_MERCHANT_ID: __vault__::secret-{{ business_unit }}/{{ environment }}/cybersource>data>merchant_id
+    ECOMMERCE_CYBERSOURCE_ACCESS_KEY: __vault__::secret-{{ business_unit }}/{{ environment }}/cybersource>data>access_key
+    ECOMMERCE_CYBERSOURCE_SECRET_KEY: __vault__::secret-{{ business_unit }}/{{ environment }}/cybersource>data>secret_key
+    ECOMMERCE_CYBERSOURCE_SOP_ACCESS_KEY: __vault__::secret-{{ business_unit }}/{{ environment }}/cybersource>data>access_key
```

Comment:
Is this supposed to be the same as `ECOMMERCE_CYBERSOURCE_ACCESS_KEY`?

Reply:
For now, yes. At least until we figure out what breaks :)